dev-lisp/sbcl/sbcl-2.1.10.22_44e0d89ff.recipe

```diff
@@ -8,16 +8,15 @@ HOMEPAGE="https://sbcl.org"
 COPYRIGHT="2002 Gerd Moellmann"
 LICENSE="BSD (2-clause)"
 REVISION="1"
-SOURCE_URI="git://github.com/sbcl/sbcl.git"
-CHECKSUM_SHA256="9cfd324f2a1eac2635188b945d10df428d1941c6"
-PATCHES="sbcl-2.1.11.patchset"
+srcGitRev="44e0d89ff14fcd8d599edd00668633c4772c26b9"
+SOURCE_URI="https://github.com/sbcl/sbcl/archive/$srcGitRev.tar.gz"
+SOURCE_DIR="sbcl-$srcGitRev"
+CHECKSUM_SHA256="b59dba5b0c4d8a599b60f29d79e29115f1dd73f3b5f305123ecf0cb1ef1a8212"
+PATCHES="sbcl-2.1.10.22_44e0d89ff.patchset"
```

Comment:
$portVersion can be used instead of hardcoding a version

Reply:
updated